spacemouse_gains:
  cartesian_velocity:
    x_axis: 0.1
    y_axis: 0.1
    z_axis: 0.1
    roll: 0.2
    pitch: 0.2
    yaw: 0.2
    lpf_const: 0.995
  cartesian_impedance:
    x_axis: 0.1
    y_axis: 0.1
    z_axis: 0.1
    roll: 0.5
    pitch: 0.5
    yaw: 0.5
    lpf_const_t: 0.995
    lpf_const_o: 0.95